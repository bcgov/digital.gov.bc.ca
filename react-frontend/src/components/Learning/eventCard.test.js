import React from 'react';
import renderer from 'react-test-renderer';
import EventCard from './eventCard';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <EventCard
        name="steve"
        description="A great description"
        eventbriteUrl="https://test.com"
        coverImgSrc="https://testIMG.com"
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
