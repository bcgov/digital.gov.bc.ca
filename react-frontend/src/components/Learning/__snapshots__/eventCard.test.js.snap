// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-iJKOzS fudOnS cardRound"
>
  <img
    className="sc-giYgFv icSVfd"
    data-testid="thumbnail"
    src="https://testIMG.com"
  />
  <div
    className="sc-bYoCmx gDuUUi"
  >
    <h5
      className="sc-egiSv kfuvTV cardTitle"
      data-testid="title"
      style={
        Object {
          "clear": "both",
          "fontSize": "25.92px",
        }
      }
    >
      steve
    </h5>
    <p
      data-testid="description"
    >
      A great description
    </p>
    <a
      className="sc-eJwXpk hGQaAH externalLink"
      href="https://test.com"
      rel="noopener noreferrer"
      style={
        Object {
          "marginTop": "auto",
        }
      }
      target="_blank"
    >
      View Details & Register
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-external-link-alt fa-w-16 "
        data-icon="external-link-alt"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={
          Object {
            "paddingLeft": "5px",
          }
        }
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </a>
  </div>
</div>
`;
